docs/CHANGELOG.rst

```diff
@@ -109,6 +109,8 @@ Changed
   process
 - Set the ``cooksCutoff`` parameter to ``FALSE`` in ``deseq.R`` tool
 - Rename 'Expressions (BCM)' to 'Dicty expressions'
+- **BACKWARD INCOMPATIBLE:** Refactor ``trimmomatic-single`` and ``trimmomatic-paired``
```

Comment:
@JureZmrzlikar, @JenkoB, you placed the CHANGELOG entry completely wrong here. Please, pay more attention!

Reply:
Sorry! Will be more careful next time...